engine/src/main/battlecode/world/InternalRobot.java

```diff
@@ -384,9 +384,31 @@ public void empower(int radiusSquared) {
                 conv++;
```

Comment:
Since we have some things in doubles and rounded down, it may make sense to do this here too, rather than having some things calculated at floating point and others calculated by distributing the remainder. What do you think?